packages/E2ETest/wdio.conf.js

```diff
@@ -1,130 +1,59 @@
+/**
+ * Copyright (c) Microsoft Corporation.
+ * Licensed under the MIT License.
+ * @format
+ * @ts-check
+ */
+
 require('ts-node').register({ transpileOnly: true });
 
-const baseUrl = 'https://webdriver.io';
+const fs = require('fs');
 
 exports.config = {
-  //
-  // ====================
-  // Runner Configuration
-  // ====================
-  //
   // WebdriverIO allows it to run your tests in arbitrary locations (e.g. locally or
   // on a remote machine).
   runner: 'local',
-  //
-  // =====================
-  // Server Configurations
-  // =====================
-  // Host address of the running Selenium server. This information is usually obsolete as
-  // WebdriverIO automatically connects to localhost. Also, if you are using one of the
-  // supported cloud services like Sauce Labs, Browserstack, or Testing Bot you don't
-  // need to define host and port information because WebdriverIO can figure that out
-  // according to your user and key information. However, if you are using a private Selenium
-  // backend you should define the host address, port, and path here.
-  //
-  // hostname: '172.18.5.185',
-  // port: 4444,
-  // path: '/wd/hub',
 
-  //
-  // ==================
-  // Specify Test Files
-  // ==================
   // Define which test specs should run. The pattern is relative to the directory
-  // from which `wdio` was called. Notice that, if you are calling `wdio` from an
-  // NPM script (see https://docs.npmjs.com/cli/run-script) then the current working
-  // directory is where your package.json resides, so `wdio` will be called from there.
-  //
-  specs: ['wdio/test/**/*.ts'],
-  // Patterns to exclude.
-  exclude: [
-    // 'path/to/excluded/files'
-  ],
-  //
-  // ============
-  // Capabilities
-  // ============
-  // Define your capabilities here. WebdriverIO can run multiple capabilities at the same
-  // time. Depending on the number of capabilities, WebdriverIO launches several test
-  // sessions. Within your capabilities you can overwrite the spec and exclude options in
-  // order to group specific specs to a specific capability.
-  //
-  // First, you can define how many instances should be started at the same time. Let's
-  // say you have 3 different capabilities (Chrome, Firefox, and Safari) and you have
-  // set maxInstances to 1; wdio will spawn 3 processes. Therefore, if you have 10 spec
-  // files and you set maxInstances to 10, all spec files will get tested at the same time
-  // and 30 processes will get spawned. The property handles how many capabilities
-  // from the same test should run tests.
-  //
+  // from which `wdio` was called.
+  specs: ['wdio/test/**/*.test.ts'],
```

Comment:
Nice, this should help color the .ts files between prod and test code in vscode as well right?

Reply:
I think VS might have been highlighting them before as well? Change from spec to test was mostly just for consistency with the Jest tests we have in repo.